### DISCUSSION TITLE
target #114 same solution matches less

### DISCUSSION THREAD

**Discussion by verberg** [cssbattle/forum#272] [Bugs reports]
Yesterday I got width 231 chars 97.4% match and now the same solution gets only 79.6% match.

Why? Has anything changed since yesterday?

And as I see now: there are now only two colors, not three like yesterday. The black one (#000000) ist not there anymore...

**Comment by Beowolve** [comment]
The target has changed, because it was almost impossible to get to 100%.
Clear your browser cache to see the new image.

Also I suggest joining on discord to get informed sooner :D

https://discord.gg/rvqvbcFM86

**Comment by chinchang** [comment]
As Beo mentioned, the target was updated. More here -> https://github.com/cssbattle/forum/discussions/273
